backend/views/data.py

```diff
@@ -0,0 +1,29 @@
+from flask import jsonify, request
+from models.data import Data
+from utils import app
+
+
+@app.route('/data-report/<file_id>', methods=['GET'])
+def data_report(file_id):
+    if request.method == 'GET':
+        result = Data.query.with_entities(
```

Comment:
@nuugut We should have some test code like this, these are actual endpoints which recieve request.

Reply:
unit test added